src/components/SettingsPage/sections/Tools.js

```diff
@@ -30,6 +31,7 @@ class TabProfile extends PureComponent<*, *> {
         <Box flow={2}>
           <QRCodeExporter />
         </Box>
+        <Box style={{ textAlign: 'right' }}>{liveCommonPkg.version}</Box>
```

Comment:
```
<Box align="right">
```
:p

Reply:
it doesn't work

Reply:
rollbacking. btw not very important because new design to come later.

Reply:
my bad, it's `textAlign="right"`, the `align` prop is bound to `align-items`

Reply:
nvm